common/datavalues/src/series/comparison.rs

```diff
@@ -40,6 +40,10 @@ fn coerce_cmp_lhs_rhs(lhs: &Series, rhs: &Series) -> Result<(Series, Series)> {
         return Ok((lhs.clone(), rhs.clone()));
     }
 
+    if lhs.data_type() == DataType::Null {
```

Comment:
If it's null, we can use  *DataArray::<T>::full_null(lhs.len()) *  to have full null DataArray<T>

Reply:
But the type of `lhs` is `&Series`, not `DataArray`. Please give more hints, thanks!

Reply:
`DataArray<T>` can convert into `Series` by IntoSeries Trait

So the code may look like 

```
 if lhs.data_type() == DataType::Null { 
    let array = DataArray::<T>::full_null(size);
    let rhs = array.into_series();
    return coerce_cmp_lhs_rhs(lhs, rhs);
 }
````